lib/server/worker.js

```diff
@@ -52,6 +52,16 @@ exports.create = function() {
     }
   });
 
+  server.route({
+    method: 'DELETE',
+    path: '/a/{id}',
+    config: {
+      handler: function delete(req, reply) {
+        img.delete(req.params.id).done(reply, reply);
```

Comment:
The worker is only accessible over the local network. So the only person who can tell the worker to delete something is the Profile web server.